src/lib/user-ip-address.js

```diff
@@ -0,0 +1,17 @@
+const net = require("net");
+
+module.exports = (request) => {
+  const forwarded = request.headers["forwarded"];
+  if (forwarded) {
+    const myObj = {};
```

Comment:
Lets give this object a better name - I just used myObj as a example.

Something like 
`forwardedHeaders = {}` maybe